Makefile

```diff
@@ -30,3 +61,11 @@ push:
 		docker push stackstorm/$$component:${DOCKER_TAG}; \
 		echo -e "\033[32mSuccessfully pushed \033[1mstackstorm/$$component:${DOCKER_TAG}\033[0m\033[32m Docker image for StackStorm version \033[1m${ST2_VERSION}\033[0m"; \
 	done
+ifeq ($(RELEASE_VERSION), true)
+	for image in st2 st2*; do \
+		docker push stackstorm/$$image:${MAJOR}; \
+		echo -e "\033[32mSuccessfully pushed \033[1mstackstorm/$$image:${MAJOR}\033[0m\033[32m Docker image for StackStorm version \033[1m${ST2_VERSION}\033[0m"; \
+		docker push stackstorm/$$image:${MAJOR}.${MINOR}; \
+		echo -e "\033[32mSuccessfully pushed \033[1mstackstorm/$$image:${MAJOR}.${MINOR}\033[0m\033[32m Docker image for StackStorm version \033[1m${ST2_VERSION}\033[0m"; \
+	done
+endif
```

Comment:
need new line at the end of file

Reply:
Already committed and will be pushed with the other additions later today. 

But why is this needed? Is it just about a style convention or linting rules or is there any technical reason for it?

Reply:
As I'm aware it's just best coding/styling practice, but looks like there are a bunch of technical reasons for that too.

Interesting:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Thanks, that's helpful! :+1: